server/models/ExperimentSection.ts

```diff
@@ -0,0 +1,26 @@
+import { DataTypes, Model } from "sequelize"
+import sequelize from "../utils/sequelize"
+import Section from "./Section"
+
+class ExperimentSection extends Model {}
+
+ExperimentSection.init(
+  {
+    // Model attributes are defined here
+    text: {
```

Comment:
I think an id of type uuid with the default value and as primary key would be good here as well